MapboxCoreNavigation/Feedback.swift

```diff
@@ -3,89 +3,48 @@ import Foundation
 /**
  Feedback type is used to specify the type of feedback being recorded with `NavigationEventsManager.recordFeedback(type:description:)`.
  */
-public enum FeedbackType: Int, CustomStringConvertible {
+public enum FeedbackType: CustomStringConvertible {
 
     /// Indicates general feedback. You should provide a `description` string to `NavigationEventsManager.recordFeedback(type:description:)`
     /// to elaborate on the feedback if possible.
     case general
```

Comment:
Not strictly necessary for this PR, but now that we no longer have to support Objective-C, this case could be removed in favor of making the feedback type optional wherever this enumeration is used.

Reply:
On second thought, that might not be a good idea, because the JSON representation of `general` is the string `general`; we’d want to centralize that representation in `FeedbackType.description` instead of hardcoding it elsewhere.